test/e2e/javascript.spec.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  'Basic JavaScript test' : function (client) {
+    client
+      .url(client.launch_url)
+      .waitForElementVisible('#panels a:nth-child(3)', 1000)
+      .click('#panels a:nth-child(3)')
+      .waitForElementVisible('.javascript .CodeMirror', 1000)
```

Comment:
You should be able to provide some kind of default timeout to get rid of `1000`'s everywhere.